packages/Popover/src/Popover.js

```diff
@@ -220,18 +220,47 @@ class Popover extends React.Component {
     this.updateVisibilityAndPositionState(isOpening);
   }
 
+  getBoundingClientRect(el) {
```

Comment:
this could be a hook, but that scope would be totally out of this PR. seem ok to me.